pkg/semver/find_test.go

```diff
@@ -0,0 +1,49 @@
+package semver
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFind(t *testing.T) {
+	type Test struct {
+		Name      string
+		Versions  []string
+		WantIndex int
+	}
+
+	var tests = []Test{
+		{Name: "GetFirstVersionIfValid", Versions: []string{"v1.0.1", "v0.1.1", "v0.1.0"}, WantIndex: 0},
+		{Name: "SkipVersionIfInvalid", Versions: []string{"invalid1", "invalid2", "v0.1.0"}, WantIndex: 2},
```

Comment:
shouldn't we add more - complex tests?
Eg with scrambled version numbers and all invalid tags?

Reply:
these are all taken care of by the underlying semver.Parse method and its unit tests, so no need - these error tests are simply there to test the skipping itself